contributor-growth/drafts/contributor-growth-framework/1-Motivating.md

```diff
@@ -0,0 +1,107 @@
+
+Be open about your roadmap and what you are trying to accomplish. Contributors want to be inspired. After all, you're asking them to dedicate their time and effort to your project. Be transparent about your vision and how you're planning on executing it. One important aspect about roadmap transparency is to let contributors know that it may change as the project matures. 
+
+When you discuss the roadmap, make sure it's all discussed publicly (e.g. community meetings, Slack, discourse, mailing list, GitHub issues or discussions). This will also help avoid situations in which someone submits a massive PR out of the blue which will almost certainly end up in a bad contributor experience. PRs aren't the best place to have feature discussion, so make sure you provide a platform for that discussion to happen (e.g Slack, GitHub Discussions, etc.). 
+
+Be clear and descriptive about how people can get involved and document it well. Good "first issues" and "help wanted" labels are great ways to signal how people can get started. It also shows that the maintainers have taken the time to identify how newcomers can help.  But also be explicit about what you need and communicate it where your users are (e.g. Twitter, Slack, tech blogs, docs, releases).  
+The more descriptive, the better. While telling contributors " just to jump in" may be easy, it can also be really overwhelming. Taking the time to describe what you need and how you need it is time-consuming, but it will also save you time down the line as people start work on things you need the way you specified them, reducing backs and forth. By describing the "what" and "how" of what you need, you are onboarding new contributors and empowering them to help other potential contributors. It's like a ponzi scheme, but for good. :)
+
+An adopters list lets people know who's using your project. Big company names always help instill confidence but may also signal people working at these companies that they could get more involved, maybe even become a maintainer. 
+More mature projects should take advantage of end-user committees. Generally composed of a diverse group of decision-makers, building relationships with your end-users and getting input on what they would like to see on the roadmap is incredibly valuable. 
+
+## Lowering Contribution Barriers  
+
+Motivation to contribute can be gone as quickly as it appeared. To ensure prospective contributors remain motivated, eliminate as much friction as possible. As mentioned above, each additional step represents an additional friction point and a possible drop-off. Make sure the entire contributor experience is intuitive and easy, including:
+   * Running the code locally before opening a PR (e.g. providing a Makefile and test data on top of instructions to build and run it)
```

Comment:
Suggestion: I would just say build script since every language and project uses different tools. 

Also, Makefiles are not very friendly for people on other operating systems (try getting make+bash to work on windows without also setting up a linux distribution with wsl), so it would be nice to not accidentally imply that makefiles are the best way to help new contributors.

Reply:
Do you mean changing this part? "Running the code locally before opening a PR" and then (e.g. providing a  make+bash to work on windows without also setting up a linux distribution with wsl). Sorry, not too familiar with any of these tools. Feel free to just go ahead and change directly. I trust you know a lot more about this than me :)

Reply:
I will submit a PR with my suggestions, sorry for using your PR as a todo list!